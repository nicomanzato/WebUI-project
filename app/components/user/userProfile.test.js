import React from 'react';
import UserProfile from './userProfile';
import UserProfileMock from './../../store/user/mock/userProfileMock'
import ShallowRenderer from 'react-test-renderer/shallow';

describe('user profile', () => {

  it('should render correctly', () => {
    const renderer = new ShallowRenderer();
    renderer.render(
      <UserProfile
        user={UserProfileMock}
      />
    );

    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });
});
