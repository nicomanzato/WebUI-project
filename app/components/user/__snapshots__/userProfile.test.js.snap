// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`user profile should render correctly 1`] = `
<Component
  style={
    Object {
      "flexDirection": "column",
    }
  }
>
  <Image
    source={
      Object {
        "uri": "https://pbs.twimg.com/profile_banners/2495199115/1530555192",
      }
    }
    style={
      Object {
        "height": 296.44444444444446,
        "resizeMode": "contain",
        "width": 750,
      }
    }
  />
  <Component
    style={
      Array [
        Object {
          "flexDirection": "row",
        },
        Object {
          "justifyContent": "center",
        },
      ]
    }
  >
    <Image
      source={
        Object {
          "uri": "https://pbs.twimg.com/profile_images/806223847557124096/rwtoQ0Cd_normal.jpg",
        }
      }
      style={
        Object {
          "alignSelf": "center",
          "borderRadius": 25,
          "height": 50,
          "resizeMode": "contain",
          "width": 50,
        }
      }
    />
    <Component
      style={
        Object {
          "alignSelf": "center",
          "height": 50,
          "justifyContent": "center",
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": 16,
          }
        }
      >
         
        SelecciÃ³n Argentina ðŸ‡¦ðŸ‡·
         
      </Text>
    </Component>
  </Component>
</Component>
`;
